### PR TITLE
feat(DateField): add component's description in README

### PR DESCRIPTION
**Pull request by Arucard89**
DATAUI-2000

### CODE REVIEW

File: src/components/DateField/README.md [hunk 1]
```diff
@@ -0,0 +1,378 @@
...
+
+# DateField
+
+<!--/GITHUB_BLOCK-->
+
+```tsx
+import {DateField} from '@gravity-ui/date-components';
+```
+
+`DateField` component is a versatile and convenient input field specifically designed for date entry in React applications. With an intuitive interface and easy integration, it's perfect for any form that requires date or time input, such as event schedulers, booking systems, or data-driven reports. It can be controlled (if you set `value` property) or uncontrolled (if you set `defaultValue` property). Component is uncontrolled by default.
```
[the remark this answers is not in the document]
Comment (Arucard89) [reply src/components/DateField/README.md]: c16648ab1b7c048da1b7d501eb3efb392a3df241

File: src/components/DateField/README.md [hunk 2]
```diff
@@ -0,0 +1,378 @@
...
+<DateField pin="brick-round" />
+```
+
+<!--/GITHUB_BLOCK-->
+
+## Value
+
+### Min and max value
+
+The `minValue` property allows you to specify the earliest date and time that can be entered by the user. Conversely, the `maxValue` property specifies the latest date and time that can be entered.
```
[the remark this answers is not in the document]
Comment (Arucard89) [reply src/components/DateField/README.md]: c16648ab1b7c048da1b7d501eb3efb392a3df241

File: src/components/DateField/README.md [hunk 3]
```diff
@@ -0,0 +1,378 @@
...
+`}
+>
+    <DateComponents.DateField format='LTS' />
+</ExampleBlock>
+LANDING_BLOCK-->
+
+<!--GITHUB_BLOCK-->
+
+```tsx
+<DateField placeholder="Placeholder" />
```
[the remark this answers is not in the document]
Comment (Arucard89) [reply src/components/DateField/README.md]: c16648ab1b7c048da1b7d501eb3efb392a3df241

File: src/components/DateField/README.md [hunk 4]
```diff
@@ -0,0 +1,378 @@
...
+| aria-label        | The control's `aria-label` attribute                                                                          |                   `string`                    |                           |
+| aria-labelledby   | The control's `aria-labelledby` attribute                                                                     |                   `string`                    |                           |
+| autoFocus         | The control's `autofocus` attribute                                                                           |                   `boolean`                   |                           |
+| className         | The control's wrapper class name                                                                              |                   `string`                    |                           |
+| defaultValue      | The control's default value, used when the component is not controlled                                        |                  `DateTime`                   |                           |
```
[the remark this answers is not in the document]
Comment (Arucard89) [reply src/components/DateField/README.md]: c16648ab1b7c048da1b7d501eb3efb392a3df241

File: src/components/DateField/README.md [hunk 5]
```diff
@@ -0,0 +1,378 @@
...
+| readOnly          | Whether the calendar value is immutable.                                                                      |                   `boolean`                   |          `false`          |
+| rightContent      | User`s node rendered after the input node and clear button                                                    |               `React.ReactNode`               |                           |
+| size              | The size of the control                                                                                       |           `"s"` `"m"` `"l"` `"xl"`            |           `"m"`           |
+| style             | Sets inline style for the element.                                                                            |                `CSSProperties`                |                           |
+| timeZone          | Sets the time zone                                                                                            |                   `string`                    |                           |
```
[the remark this answers is not in the document]
Comment (Arucard89) [reply src/components/DateField/README.md:381]: c16648ab1b7c048da1b7d501eb3efb392a3df241

[the remark this answers is not in the document]
Comment (korvin89) [reply src/components/DateField/README.md:381]: Could u add [this link](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List) here?

File: src/components/DateField/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (korvin89) [reply src/components/DateField/README.md]: There is no need in "Optional" word here, because we have column with this information

**Comment by Arucard89** [comment]
Made all fixes

**Review by korvin89 — APPROVED** [review]

**Pull request merged**